_dprhtml/js/grammar.js

```diff
@@ -418,31 +413,28 @@ function conjugateIrrNoun(word) {
 }
 
 function conjugateIrrVerb(word,which) {
-  var yto = DPR_G.yt[word];
   var out = '';
 
   var outword = word.replace(/\`/g, '"');
   outword = outword.replace(/,/g, '.');
-  if(yto[5].search(/\.[āīū],/) > -1 || yto[5].search(/\.[āīū]$/) > -1) outword = outword+outword.charAt(outword.length-1);
+  if(CPED.getInflectionInfo(word).search(/\.[āīū],/) > -1 || CPED.getInflectionInfo(word).search(/\.[āīū]$/) > -1) outword = outword+outword.charAt(outword.length-1);
   outword = DPR_translit_mod.toUni(outword);
 
-  var stem = yto[8];
```

Comment:
This variable is not referenced anywhere, as indicated by the built-in VSCode JavaScript linter.